src/Scene.jl

```diff
@@ -9,7 +10,7 @@
     function Scene()
         this = new()
 
-        this.camera = C_NULL
+        this.camera = C_NULL # why use C_NULL not nothing?
```

Comment:
This is also a lack of knowledge of julia. I think the reason why I use it so much is from using the SDL package. Is using Nothing faster? Definitely need to audit the usage of this

Reply:
I don't think it's faster, just idiomatic. I think C_NULL is really just for interfacing with ccalls. Nothing represents "Nothing here for now, I'll fill it in later"

Reply:
Gotcha. That might have to be a separate issue for some standardization. I don't know everywhere we check for this value in other files.